api/src/pcapi/core/mails/transactional/pro/venue_needs_picture.py

```diff
@@ -0,0 +1,22 @@
+from pcapi.core import mails
+from pcapi.core.mails import models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.core.offerers import models as offerer_models
+from pcapi.utils import urls
+
+
+def get_permanent_venue_needs_picture_data(venue: offerer_models.Venue) -> models.TransactionalEmailData:
+    return models.TransactionalEmailData(
+        template=TransactionalEmail.VENUE_NEEDS_PICTURE.value,
+        params={
+            "VENUE_NAME": venue.publicName or venue.name,
```

Comment:
[détail] 

```suggestion
            "VENUE_NAME": venue.common_name,
```

(idem dans le test)

Reply:
Bon à savoir !